contrib/pg_tde/documentation/docs/replication.md

```diff
@@ -1,4 +1,104 @@
-# Replication
+# Streaming Replication with tde_heap
 
-<!-- To add information on Replication affecting pg_tde for customers and how it interacts with it
-Also, don't forget to add it to TOC>
\ No newline at end of file
+This section outlines how to set up PostgreSQL streaming replication when the `pg_tde` extension (specifically the `tde_heap` access method) is enabled on the primary server.
+
+The following steps assume:
+
+* You have configured a global key provider for both **primary** and **standby** (see [Configure Key Management (KMS)](global-key-provider-configuration/index.md)).
```

Comment:
you don't have to configure it for the standby, it's automtatic (other than that the certificate files has to be accessible for the standby too, and you have to add pg_tde to the shared preload libraries)